app/views/posts/index.html.erb

```diff
@@ -1 +1,40 @@
-<h1> Shows posts per user </h1>
\ No newline at end of file
+<div class="container bg-gray-900 text-white">
+  <div class="container flex justify-between items-center p-8">
+    <div class="photo-container w-48 h-48 rounded-full overflow-hidden">
+      <img src="<%= @user.photo %>" alt="User Photo" class="object-cover">
+    </div>
+    <div class="info-container border-2 w-2/4 p-7 rounded-lg">
+      <h2 class="text-2xl font-bold"><%= @user.name%></h2>
+      <p class="text-gray-600 text-right">Number of Posts: 10</p>
+    </div>
```

Comment:
- [ ] You've done an excellent job, while I added 22posts, I observed that the **number of post** in the `post_index` page does not tally; this is because you've hardcoded it. Kindly implement it such that the appropriate number of post(s) are displayed. ℹ️ 

https://github.com/Rebelzob/blog-app/assets/103386183/cdf612c9-53ba-4b3f-920f-494d40162827

Reply:
Thanks for the review, I just made the changes!